lib/PhpFlo/Component.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/*
+ * This file is part of the phpflo/phpflo package.
+ *
+ * (c) Henri Bergius <henri.bergius@iki.fi>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace PhpFlo;
+
+use PhpFlo\Common\ComponentInterface;
+use PhpFlo\Interaction\PortRegistry;
+
+/**
+ * Class Component
+ *
+ * @package PhpFlo
+ * @author Henri Bergius <henri.bergius@iki.fi>
+ */
+class Component implements ComponentInterface
+{
+    /**
+     * @var PortRegistry
+     */
+    private $_inPorts = null;
```

Comment:
You're absolutely right! Damn. That's from my try to use magic methods to initialize the PortRegistry for in/out ports. But PSR-2... that's outright embarrassing to miss that O_o